pkg/controllers/externalsecret/externalsecret_controller_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("Kind=secret existence logic", func() {
 		})
 	}
 })
-var _ = Describe("ExternalSecret controller", func() {
+var _ = Describe("ExternalSecret controller", Serial, func() {
```

Comment:
`go test -race` reports an issue here. I didn't take a look too deeply. Because it has been stable for a very long time i'd just put this here, that seems to suppress the data race report.